.github/workflows/build.yml

```
@@ -51,6 +51,11 @@ jobs:
       fail-fast: false
     name: "Build ${{ matrix.variant }}-${{ matrix.arch }}"
     steps:
+      - name: Random delay
+        run: |
+          delay=$((1 + $RANDOM % 30))
```

nit: perhaps this to avoid bias:
```suggestion
          delay=$((1 + $RANDOM % 32))
```